cmd/main.go

```diff
@@ -28,13 +28,15 @@ import (
 
 func main() {
 	var (
-		endpoint                = flag.String("endpoint", "unix://tmp/csi.sock", "CSI Endpoint")
-		version                 = flag.Bool("version", false, "Print the version and exit")
-		efsUtilsCfgDirPath      = flag.String("efs-utils-config-dir-path", "/etc/amazon/efs/", "The path to efs-utils config directory")
-		efsUtilsStaticFilesPath = flag.String("efs-utils-static-files-path", "/etc/amazon/efs-static-files/", "The path to efs-utils static files directory")
-		volMetricsOptIn         = flag.Bool("vol-metrics-opt-in", false, "Opt in to emit volume metrics")
-		volMetricsRefreshPeriod = flag.Float64("vol-metrics-refresh-period", 240, "Refresh period for volume metrics in minutes")
-		volMetricsFsRateLimit   = flag.Int("vol-metrics-fs-rate-limit", 5, "Volume metrics routines rate limiter per file system")
+		endpoint                 = flag.String("endpoint", "unix://tmp/csi.sock", "CSI Endpoint")
+		version                  = flag.Bool("version", false, "Print the version and exit")
+		efsUtilsCfgDirPath       = flag.String("efs-utils-config-dir-path", "/etc/amazon/efs/", "The path to efs-utils config directory")
+		efsUtilsStaticFilesPath  = flag.String("efs-utils-static-files-path", "/etc/amazon/efs-static-files/", "The path to efs-utils static files directory")
+		volMetricsOptIn          = flag.Bool("vol-metrics-opt-in", false, "Opt in to emit volume metrics")
+		volMetricsRefreshPeriod  = flag.Float64("vol-metrics-refresh-period", 240, "Refresh period for volume metrics in minutes")
+		volMetricsFsRateLimit    = flag.Int("vol-metrics-fs-rate-limit", 5, "Volume metrics routines rate limiter per file system")
+		deleteAccessPointRootDir = flag.Bool("delete-access-point-root-dir", false,
```

Comment:
what do we think about default true? I don't want users to be surprised by efs usage since usually delete means delete data. I considered default false before to "gate" the feature, but there is no point, the whole driver is beta.

Reply:
I think we should keep this as false and not delete data by default and not delete data accidentally. This also ensures the administrator/user is aware that by turning on this feature, data is deleted.

Reply:
well, everywhere else in kubernetes when PVC is deleted then the intention is that the data and backing volume are deleted. I am still convinced we need to default true but I will let @jqmichael tie break. nfs client provisioner had a similar issue where the default behaivor was "archiveOnDelete" and that conflicted with user expectations. If we commit to defaulting false now, it will be even harder to change it in the future.

Reply:
I was also suprised by this, but neutral about the default.  It would be great to add strong documentation about how different this is than expectations with other drivers and the downsides (infinite accumulating data) in involves that will require out of bound maintenance.